### ISSUE TITLE
can I specify image of mysqld-exporter and so on?

### ISSUE THREAD

**Issue by cai2yy** [presslabs/mysql-operator#531]
I try to deploy mysql-operator in an off-line enviroment, and prom/mysqld-exporter:v0.11.0 image pull failed.
Where can I find the field to specify image of exporte?

**Comment by AMecea** [comment]
Yes, there is a command-line argument that lets you specify the metrics exporter image, [`--metrics-exporter-image`](https://github.com/presslabs/mysql-operator/blob/c0ac4b6918c145eb115ed90ea31c4c6cac089e32/pkg/options/options.go#L119)

Just add in your `values.yaml` the following:
```yaml
extraArgs:
 - --metrics-exporter-image=<your image>
```
